static/src/javascripts/projects/common/views/commercial/creatives/logo-ad-feature.html

```diff
@@ -3,8 +3,4 @@ <h3 class="ad-slot--paid-for-badge__header"><%=logoFeatureLabel%></h3>
     <a href="<%=clickMacro%><%=logoUrl%>" class="ad-slot--paid-for-badge__link">
         <img class="ad-slot--paid-for-badge__logo" src="<%=logoImage%>" />
     </a>
-    <% if (!guardian.config.switches.newCommercialContent) { %>
```

Comment:
This file is only used for paid content so it's safe to get rid of the 'about this' link.
Also it's been switched off for weeks.